consts.go

```diff
@@ -27,3 +27,14 @@ const (
 
 	DefaultClientName = "centralDogmaClient"
 )
+
+const (
+	urlPartProjects = "projects"
+	urlPartRepos    = "repos"
+	urlPartContents = "contents"
+	urlPartCommits  = "commits"
```

Comment:
How about removing `urlPart` and just using the component like `projects` and `repos` as we do in [upstream](https://github.com/line/centraldogma/blob/master/common/src/main/java/com/linecorp/centraldogma/internal/api/v1/HttpApiV1Constants.java#L29)?

Reply:
SGTM. Let me change the const name 👍